Automattic/wp-calypso
# Ad-Tracking: Load Parsely Script

### Summary
Load Parsely tracking script in Calypso and record the purchase event

### Test Plan

#### Prerequisites
1. Make sure that browser doNotTrack setting is disabled. For example, in chrome, [follow these instructions to disable](https://support.google.com/chrome/answer/2790761?hl=en&co=GENIE.Platform%3DDesktop).
2. Enable the "Share information with our analytics tool about your use of services while logged in to your WordPress.com account" toggle in /me under the Privacy tab
3. Go to [config/development.json](https://github.com/Automattic/wp-calypso/blob/trunk/config/development.json) and change `ad-tracking` and `cookie-banner` to `true`.
4. Add define( 'USE_STORE_SANDBOX', true ); to wp-content/mu-plugins/0-sandbox.php to sandbox the store so you can checkout with a test credit card
5. Sandbox public-api.wordpress.com


#### Testing
1. Checkout this branch
2. Run `yarn` & `yarn start`
3. Go to http://calypso.localhost:3000/start and accept the cookie banner
4. Keep the  Network tab open in DevTools and sort the requests by `Image` and filter by `?rand`
5. Go through the Checkout flow
6. Complete the purchase with a test credit card and verify we send a conversion to Parsely with the current order id.

#### Video

https://github.com/Automattic/wp-calypso/assets/20927667/ee59228f-b68d-479c-83ab-256b7b1211bc
